mediaplayerbackend.cpp

```diff
@@ -41,8 +41,22 @@
 ****************************************************************************/
 
 #include "mediaplayerbackend.h"
+#include "logging.h"
 
 #include <QDebug>
+#include <QImage>
+
+#ifndef QTIVI_NO_TAGLIB
```

Comment:
A bit confusing to use ifndef in conjunction with _NO_. I would prefer
`#ifdef QTIVI_TAGLIB`
and then include the files required.

Reply:
It depends on the default state: taglib is normally present and should as it is shipped with qtivi as a 3rd-party component, but some users might want to disable it for whatever reason.

Reply:
Ok